Sources/BSWInterfaceKit/SwiftUI/DataSource/InfiniteScrollingDataSource.swift

```diff
@@ -47,23 +47,21 @@ open class InfiniteScrollingDataSource<ListItem: Identifiable>: ObservableObject
     
     /// MARK: Private
     
-    @MainActor
-    private func loadMoreContent() async throws {
+    public func loadMoreContent() async throws {
         guard case .canLoadMorePages(let currentPage) = state else {
             return
         }
-        withAnimation {
+        let stateAnimation = Animation.easeInOut(duration: 0.3)
```

Comment:
Given that we now don't need to wait for one animation to end before the other runs, we can just use the default animation